docs/concepts/what-is-a-schedule.md

```diff
@@ -0,0 +1,197 @@
+The time zone definition will be loaded on the Temporal Server Worker Service from either disk or the fallback embedded in the binary.
+
+For more operational control, embed the contents of the time zone database file in the Schedule Spec itself.
+(Note: this isn't currently exposed in tctl or the web UI.)
+
+### Pausing
+
+A Schedule may be Paused.
+When a Schedule is Paused, the Spec has no effect.
+However, you can still force manual actions, see [trigger immediately](/tctl/schedule/trigger).
+
+To assist communication among developers and operators, there's a “notes” field that can be updated on pause/resume to store an explanation for the current state.
+
+### Limiting number of Actions
+
+A Schedule may be limited to a certain number of scheduled Actions (i.e. not trigger immediately).
+After that it will act as if it was Paused.
+
+### Policies
+
+A Schedule supports a set of Policies that enable customizing behavior.
+
+#### Overlap Policy
+
+The Overlap Policy controls what happens when it is time to start a Workflow Execution but a previously started Workflow Execution is still running.
+The following options are available:
+
+- `Skip`: **Default**.
+  Nothing happens – the Workflow Execution is not started.
+- `BufferOne`: Starts the Workflow Execution as soon as the current one completes. The buffer is limited to one.
+  If another Workflow Execution is supposed to start, but there is already one in the buffer, only the one in the buffer eventually starts.
+- `BufferAll`: Allows an unlimited number of Workflows to buffer.
+  They are started sequentially.
+- `CancelOther`: Cancels the running Workflow Execution, and then starts the new one after the old one completes cancellation.
+- `TerminateOther`: Terminates the running Workflow Execution and starts the new one immediately.
+- `AllowAll` Starts any number of concurrent Workflow Executions.
+  With this policy (and only this policy), there may be more than one Workflow Execution, started by the Schedule, running simultaneously.
+
+#### Catchup Window
+
+It may happen that the Temporal Cluster itself is down or unavailable at the time when a Schedule should take an Action.
+When it comes back up, the Catchup Window controls which missed actions should be taken at that point.
+The default is one minute, which means that the Schedule will attempt to take any actions that wouldn't be more than one minute late.
+An outage that lasts longer than the catchup window could lead to missed actions (But you can always Backfill).
+
+#### Pause-on-failure
+
+If this policy is set, a Workflow Execution started by a Schedule that ends with a failure or timeout (but not Cancellation or Termination) will cause the Schedule to automatically pause.
+
+Note that with the `AllowAll` overlap policy, this pause might not apply to the next Run, because the next run might have started before the failed run finished.
+It applies only to Runs that were Scheduled to start after the failed Run finished.
```

Comment:
Styling question: Should "Overlap Policy" be init-capped here to match the earlier section? (Your answer to my question about "Pause-On-Failure" also affects this decision.)
```suggestion
Note that with the `AllowAll` overlap policy, this pause might not apply to the next run, because the next run might have started before the failed run finished.
It applies only to runs that were scheduled to start after the failed run finished.
```

Reply:
Yes, Overlap Policy should be init-capped.
I changed Run to Workflow Execution to be more precise.